tests/bdd/test_config.ini

```diff
@@ -0,0 +1,5 @@
+# VMs used to run, in effect, "nvme connect" on
+[load_generators]
+vm1 = 127.0.0.1
```

Comment:
oh, do we use a vm?

Reply:
I've removed this file. It was copied from Mayastor and isn't needed.